<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlanInfoDialog</class>
 <widget class="QWidget" name="PlanInfoDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>550</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>550</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>550</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(64, 115, 175);</string>
  </property>
  <widget class="QWidget" name="widget_PlanInfo" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>361</width>
     <height>511</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(219, 226, 239);
border-radius:15px;
border-style: solid;
border-width: 1px;
border-color:rgb(0, 0, 0);</string>
   </property>
   <widget class="QLabel" name="label_Subject">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Gentium Book Basic&quot;;
color:black;
background-color:rgb(219, 226, 239);
border-width:0px;</string>
    </property>
    <property name="text">
     <string>Subject:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_PlanInfo">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 20pt &quot;Gentium Book Basic&quot;;
color:black;
background-color:rgb(219, 226, 239);
border-width:0px;</string>
    </property>
    <property name="text">
     <string>Plan info</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Link">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Gentium Book Basic&quot;;
color:black;
background-color:rgb(219, 226, 239);
border-width:0px;
</string>
    </property>
    <property name="text">
     <string>Link:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_SubjectName">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>341</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 255, 255);
border-radius:15px;
border-style: solid;
border-width: 1px;
border-color:rgb(0, 0, 0);
font: 14pt &quot;Gentium Book Basic&quot;;
color:black;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_Teacher">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Gentium Book Basic&quot;;
color:black;
background-color:rgb(219, 226, 239);
border-width:0px;</string>
    </property>
    <property name="text">
     <string>Teacher:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_TeacherName">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>341</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 255, 255);
border-radius:15px;
border-style: solid;
border-width: 1px;
border-color:rgb(0, 0, 0);
font: 14pt &quot;Gentium Book Basic&quot;;
color:black;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_Room">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>121</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Gentium Book Basic&quot;;
color:black;
background-color:rgb(219, 226, 239);
border-width:0px;</string>
    </property>
    <property name="text">
     <string>Room:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_RoonNumber">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>341</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 255, 255);
border-radius:15px;
border-style: solid;
border-width: 1px;
border-color:rgb(0, 0, 0);
font: 14pt &quot;Gentium Book Basic&quot;;
color:black;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_LinkName">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>341</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 255, 255);
border-radius:15px;
border-style: solid;
border-width: 1px;
border-color:rgb(0, 0, 0);
font: 14pt &quot;Gentium Book Basic&quot;;
color:black;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_Repeating">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Gentium Book Basic&quot;;
color:black;
background-color:rgb(219, 226, 239);
border-width:0px;
</string>
    </property>
    <property name="text">
     <string>Repeating:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_RepeatingName">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>341</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 255, 255);
border-radius:15px;
border-style: solid;
border-width: 1px;
border-color:rgb(0, 0, 0);
font: 14pt &quot;Gentium Book Basic&quot;;
color:black;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Edit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>450</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(18, 46, 79);
border-radius:15px;
border-style: solid;
border-width: 1px;
border-color:rgb(0, 0, 0);
font: 75 20pt &quot;Gentium Book Basic&quot;;
color:white;</string>
    </property>
    <property name="text">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Delete">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>450</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(18, 46, 79);
border-radius:15px;
border-style: solid;
border-width: 1px;
border-color:rgb(0, 0, 0);
font: 75 20pt &quot;Gentium Book Basic&quot;;
color:white;</string>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
